src/sentry/workflow_engine/types.py

```diff
@@ -12,13 +12,22 @@
 T = TypeVar("T")
 
 
+class ActionType(StrEnum):
+    NOTIFICATION = "SendNotificationAction"
+    TRIGGER_WORKFLOW = "TriggerWorkflowAction"
```

Comment:
FYI, these are already defined here; https://github.com/getsentry/sentry/blob/0fa5656591c652d172dcc51b6299cd7d282f60aa/src/sentry/workflow_engine/models/action.py#L32-L34 -- we may want to even move this to be something similar to that data_sources PR i linked to, but for now we can probs reuse the `Action.Type` class.